documentation/modules/osh-adding-source-provider.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="osh-adding-source-provider_{context}"]
+= Adding an {osp} source provider
+
+You can add an {osp} source provider by using the {project-short} web console.
+
+[IMPORTANT]
+====
+Migration using {osp} source providers is a Technology Preview feature only. Technology Preview features
+are not supported with Red Hat production service level agreements (SLAs) and
+might not be functionally complete. Red Hat does not recommend using them
+in production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during
+the development process.
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+[NOTE]
+====
+Migration using {osp} source providers only supports VMs that use only Cinder volumes.
+====
+
+.Procedure
+
+. In the {project-short} web console, click *Providers*.
```

Comment:
this needs to align with the changes in #357 (also above where `{project-short} web console` appears)